### PR TITLE
gatk4-r-deps

### PR DESCRIPTION
**Pull request by lecorguille**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

___

According to this https://software.broadinstitute.org/gatk/documentation/article?id=11049
> R dependencies
>Some of the GATK tools produce plots using R, so if you want to get the plots you'll need to have R and Rscript installed, as well as these R libraries: gsalib, ggplot2, reshape, gplots,

### CODE REVIEW

**Comment by lecorguille** [comment]
@BiocondaBot please add label

File: recipes/gatk4/meta.yaml [hunk 1]
```diff
@@ -11,13 +11,17 @@ source:
...
   noarch: generic
-  number: 0
+  number: 1
 
 requirements:
   host:
   run:
     - openjdk >=8,<9
     - python
+    - r-gplots
```
Comment (bgruening) [inline recipes/gatk4/meta.yaml]: can you please add r-base as well

Comment (lecorguille) [reply recipes/gatk4/meta.yaml]: Thanks!
Done
I though that it was implicite with any r package

Comment (bgruening) [reply recipes/gatk4/meta.yaml]: It is impplicit, but conda has some magic that is done if you specify the main scripting language.

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**